src/openklant/management/commands/migrate_to_v2.py

```diff
@@ -0,0 +1,475 @@
+    )
+
+    partijen_path = reverse(
+        "klantinteracties:partij-list",
+        kwargs=dict(version=settings.REST_FRAMEWORK["DEFAULT_VERSION"]),
+    )
+
+    dummy_token = _generate_dummy_token()
+    openklant_client = OpenKlantClient(url, dummy_token)
+    generic_client = Client()
+    created_klanten = []
+
+    for klant in klanten:
+        digitaal_adres = klant.to_digitaal_adres()
+        digitaal_adres_ref = None
+
+        if digitaal_adres:
+            _data = openklant_client.create(
+                digitaal_addressen_path, digitaal_adres.dict()
+            )
+
+            if not isinstance(_data, dict):
+                logger.error(
+                    f"Unknown data received creating a digitaal adres: {_data}. "
+                    "Skipping klant."
+                )
+                continue
+
+            digitaal_adres_ref = _data.get("uuid")
+
+        if klant.subject and not klant.subject_identificatie:
+            subject_identificatie = generic_client.retrieve(klant.subject)
+
+            if not isinstance(subject_identificatie, dict):
+                logger.error(
+                    f"Unknown data received receiving a subject: {subject_identificatie}. "
+                    "Skipping klant."
+                )
+
+            klant.subject_identificatie = subject_identificatie
+
+        partij = klant.to_partij(digitaal_adres=digitaal_adres_ref)
+
+        if not partij:
+            logger.error("Unable to create partij, skipping klant..")
+            continue
+
+        response_data = openklant_client.create(partijen_path, partij.dict())
+
+        if response_data and "url" in response_data:
+            created_klanten.append(response_data["url"])
+
+    return created_klanten
+
+
+def _generate_dummy_token() -> str:
+    token_auth, _ = TokenAuth.objects.get_or_create(application="Migration application")
+    return token_auth.token
+
+
+class Command(BaseCommand):
+    def add_arguments(self, parser: CommandParser) -> None:
+        parser.add_argument(
+            "v1_url",
+            type=str,
+            metavar="https://example.openklant.nl",
+            help="URL of the Klanten API",
+        )
+
+        parser.add_argument(
+            "v2_url",
+            type=str,
+            metavar="https://example.klantinteracties.nl",
+            help="URL of the Klantinteracties API",
+        )
+
+    def handle(self, *args: Any, **options: Any) -> str | None:
+        access_token = os.getenv("ACCESS_TOKEN")
+        v1_url = options["v1_url"]
+        v2_url = options["v2_url"]
+
+        url_validator = URLValidator(schemes=["http", "https"])
+
+        for url in (v1_url, v2_url):
+            try:
+                url_validator(url)
+            except ValidationError as e:
+                return (
+                    f"Invalid URL(s) detected: {str(e.message)}. See "
+                    "migrate_to_v2 --help for correct usage."
+                )
```

Comment:
We could maybe raise a `CommandError` here to make the command return an error code as well, but it's a minor thing